core/src/main/java/com/google/zxing/pdf417/decoder/DecodedBitStreamParser.java

```diff
@@ -172,35 +179,58 @@ private static int decodeMacroBlock(int[] codewords, int codeIndex, PDF417Result
     codeIndex = textCompaction(codewords, codeIndex, fileId);
     resultMetadata.setFileId(fileId.toString());
 
-    switch (codewords[codeIndex]) {
-      case BEGIN_MACRO_PDF417_OPTIONAL_FIELD:
-        codeIndex++;
-        int[] additionalOptionCodeWords = new int[codewords[0] - codeIndex];
-        int additionalOptionCodeWordsIndex = 0;
+    while (codeIndex < codewords[0]) {
+        switch (codewords[codeIndex]) {
```

Comment:
The project uses 2-space indent

Reply:
I added also an .editorconfig, that way i didn't had to fix the identation manually.